### DISCUSSION TITLE
monitor with wazuh disabled user changed to enable in active directory

### DISCUSSION THREAD

**Discussion by tzacha898** [wazuh/wazuh#15839] [General]
Hello,
I want to create a rule to identify a situation when an already disabled user account changed to enable in my DCs from the active directory on my environment , the problem is, when a user is created I get the same alert when the user changed to enable , because users are automatically created in enable mode.
my rule is based on windows eventIDs logs, and my rule seems like this for a while.

  <rule id="260109" level="10">
    <if_sid>60103</if_sid>
    <field name="win.system.eventID">^4722$</field>
    <description>disabled user account changed to enable</description>
  </rule>  
 

if somebody had the same issue I would appreciate your help, or any ideas as well, would be very helpful.

Thank you.

**Comment by mdiego92** [comment]
Hello @tzacha898,

You could try overwriting the existing rule for the event and adding a new one to match your requirement:

**1) Add the following rules to your custom ruleset or to the `/var/ossec/etc/rules/local_rules.xml` file:**

```
<group name="windows,windows_security,">

  <rule id="60109" level="8" overwrite="yes">
    <if_sid>60103</if_sid>
    <field name="win.system.eventID">^624$|^4720$</field>
    <options>no_full_log</options>
    <description>User account created.</description>
    <mitre>
      <id>T1098</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>

  <rule id="100111" level="8">
    <if_sid>60103</if_sid>
    <field name="win.system.eventID">^626$|^4722$</field>
    <options>no_full_log</options>
    <description>User account enabled.</description>
    <mitre>
      <id>T1098</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>

</group>
```

**2) Restart the Wazuh Manager.**

Article for reference: https://documentation.wazuh.com/current/user-manual/ruleset/ruleset-xml-syntax/rules.html

Please let me know if that worked for you!

**Reply by tzacha898** [reply]
Hi Diego,
I have tried that, and unfortunately it didn't work.
I have to mention that when a user is created he is become enabled as well so I still gets alerts on my wazuh manager, just about "user account enable" but it is for the same scenario as "disabled account changes to enable".

I have tried to corelate the  <if_sid> statement in the 100111 rule,  just for a scenario when account is disabled - existing rule 60111 but it didn't work. do you have any idea why?

If you have other ideas it will be blessed.
Thank you for your help I appreciate it so much!

**Reply by mdiego92** [reply]
Hello @tzacha898,

Could you please check the windows event logs and verify if there is any field that might help us to match a custom rule to differentiate the events?

Thank you very much!

**Reply by tzacha898** [reply]
Hello @mdiego92 , when we disable account and than enable him we get the following in the event viewer.

there are 2 events of 4738 which means account is changed.
there are 1 event of 4725 which means account is disabled.
and 1 event of 4722 which means account is enabled.

when we create a new account, at first we have 1 event of 4720, which means user is created.
than we have a 3 events of 4738.
an 1 of 4722- enable. maybe we can try to configure the time of the 4738 to isolate the situation just for enable?

inside the logs there is no such a different data, but the attribute of the UAC attribute changes: 


Changed Attributes:
	SAM Account Name:	-
	Display Name:		-
	User Principal Name:	-
	Home Directory:		-
	Home Drive:		-
	Script Path:		-
	Profile Path:		-
	User Workstations:	-
	Password Last Set:	-
	Account Expires:		-
	Primary Group ID:	-
	AllowedToDelegateTo:	-
	Old UAC Value:		0x15
	New UAC Value:		0x11
	User Account Control:	
		**'Password Not Required' - Disabled**


maybe this is another option we can play with,
Thank for your help.

Tzach

**Reply by mdiego92** [reply]
Hello @tzacha898,

Yes, you could try modifying the previous rule:

```
  <rule id="100111" level="8">
    <if_sid>60103</if_sid>
    <field name="win.system.eventID">^4738$</field>
    <options>no_full_log</options>
    <description>User account enabled.</description>
    <mitre>
      <id>T1098</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>
```

Probably you will need to remove the 4722 eventID from the <field name="win.system.eventID">.

Please let me know if that worked!

**Reply by tzacha898** [reply]
I need to remove the 4722 eventID where from? @mdiego92

**Reply by tzacha898** [reply]
Hi @mdiego92 ,
I still get the same log when I create a new user as well as disable to enable.
there is an option to count the eventID 4738 so I can separate it just for maximum two times to get an alert?
thank you.

**Reply by mdiego92** [reply]
Hello @tzacha898,

You will need to add the following rule to overwrite the default one that triggers with the 4722 eventID:

`  <rule id="18110" level="8" overwrite="yes">
    <if_sid>18104</if_sid>
    <id>^624$|^626$|^4720$</id>
    <description>Windows: User account enabled or created.</description>
    <mitre>
      <id>T1098</id>
    </mitre>
    <group>adduser,account_changed,</group>
    <group>pci_dss_8.1.2,pci_dss_10.2.5,gpg13_7.10,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_IA.4,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>`

With this configuration, this rule will trigger only when receiving eventID 624, 626, and 4720 only.

**Reply by tzacha898** [reply]
Hello @mdiego92 
This is what you want me to try?


  <rule id="60109" level="8" overwrite="yes">
    <if_sid>18104</if_sid> 
    <id>^624$|^626$|^4720$</id> 
    <description>Windows: User account enabled or created.</description> 
    <mitre> 
      <id>T1098</id> 
    </mitre> 
    <group>adduser,account_changed,</group> 
    <group>pci_dss_8.1.2,pci_dss_10.2.5,gpg13_7.10,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_IA.4,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group> 
  </rule>
   
  <rule id="100111" level="8">
    <if_sid>60103</if_sid>
    <field name="win.system.eventID">^4722$</field>
    <options>no_full_log</options>
    <description>User account enabled.</description>
    <mitre>
      <id>T1098</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>

**Reply by mdiego92** [reply]
Hello @tzacha898,

I'm not able to properly see your last response, could you please send it again with code formatting?
Thank you very much!

**Reply by mdiego92** [reply]
Hello @tzacha898,

Please let me know if you have any updates on this. I'll be glad to help you.

**Reply by tzacha898** [reply]
Hello @mdiego92 
this is what you want me to try ?


```

   <rule id="60109" level="8" overwrite="yes">
    <if_sid>18104</if_sid> 
     <id>^624$|^626$|^4720$</id> 
     <description>Windows: User account enabled or created.</description> 
     <mitre> 
       <id>T1098</id> 
     </mitre> 
    <group>adduser,account_changed,</group> 
     <group>pci_dss_8.1.2,pci_dss_10.2.5,gpg13_7.10,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_IA.4,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group> 
   </rule>
    
   <rule id="100111" level="8">
     <if_sid>60103</if_sid>
     <field name="win.system.eventID">^4738$</field>
     <options>no_full_log</options>
    <description>User account changed.</description>
     <mitre>
       <id>T1098</id>
     </mitre>
     <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
 </rule>
-  

```

**Reply by mdiego92** [reply]
Hello @tzacha898,

You will need to overwrite the current rule 18110 to make it work. The configuration to test it out should be something like this:

```
  <rule id="18110" level="8" overwrite="yes">
    <if_sid>18104</if_sid>
    <id>^624$|^626$|^4720$</id>
    <description>Windows: User account enabled or created.</description>
    <mitre>
      <id>T1098</id>
    </mitre>
    <group>adduser,account_changed,</group>
    <group>pci_dss_8.1.2,pci_dss_10.2.5,gpg13_7.10,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_IA.4,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>

  <rule id="100111" level="8">
    <if_sid>60103</if_sid>
    <field name="win.system.eventID">^4738$</field>
    <options>no_full_log</options>
    <description>User account enabled.</description>
    <mitre>
      <id>T1098</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>
```

Please let me know the results. I'll wait for your update.
Thank you!

**Reply by tzacha898** [reply]
Hello @mdiego92 
it still doesn't work for both of the scenarios.
I didn't get any alerts at all with those rules.
Thank You

**Reply by mdiego92** [reply]
Hello @tzacha898,

Let me perform some tests in my lab and I'll provide an update on this as soon as I can.
Thank you for your patience.

**Comment by tzacha898** [comment]
Sure, I will let you know.

**Comment by tzacha898** [comment]
Sure, take your time.

**Reply by mdiego92** [reply]
Hello @tzacha898,

I'm sending you the group of rules that you will need to add:

```
<group name="windows,windows_security,">

  <rule id="60109" level="8" overwrite="yes">
    <if_sid>60103</if_sid>
    <field name="win.system.eventID">^624$|^4720$</field>
    <options>no_full_log</options>
    <description>User account created.</description>
    <mitre>
      <id>T1098</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>
  
  <rule id="60111" level="8" overwrite="yes">
    <if_sid>60103</if_sid>
    <field name="win.system.eventID">^630$|^4726$</field>
    <options>no_full_log</options>
    <description>User account deleted.</description>
    <mitre>
      <id>T1098</id>
      <id>T1531</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>

  <rule id="100111" level="8">
    <if_sid>60103</if_sid>
    <field name="win.system.eventID">^626$|^4722$</field>
    <options>no_full_log</options>
    <description>User account enabled.</description>
    <mitre>
      <id>T1098</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>

  <rule id="100112" level="8">
    <if_sid>60103</if_sid>
    <field name="win.system.eventID">^629$|^4725$</field>
    <options>no_full_log</options>
    <description>User account disabled.</description>
    <mitre>
      <id>T1098</id>
      <id>T1531</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>
  
</group>
```

When creating an account it will trigger both rules, but when activating a disabled account it will trigger only the new one. I did the same with the rules that delete and disable users. Hope it works!

Sorry for the late response. Please let me know if you have any remaining questions!

**Comment by tzacha898** [comment]
Thanks, I'll check them out.

**Reply by mdiego92** [reply]
Hello @tzacha898,
Were you able to test the new rules? Please let me know if it worked.

Kind regards.

**Reply by tzacha898** [reply]
Hello @mdiego92 
I'm testing it right now so I will let you know as soon as possible

**Reply by tzacha898** [reply]
Hi @mdiego92 
It still doesn't work ,
I don't get any alerts of any of the rule you suggested

**Reply by mdiego92** [reply]
Hello @tzacha898,

Could you please share with me a screenshot of how you have the new rules set up? I have tested them and had no issues.
I forgot to ask, did you remove the previous ones after adding these last four rules?

Thank you!

**Reply by tzacha898** [reply]
Hi @mdiego92 
sorry for the late response I don't have my lab on hand.
I will send it to you as soon as possible 

Regards.

**Reply by tzacha898** [reply]
Hi @mdiego92 
I still have the same alert when i create a new users as well as turn to enable a disabled user account..
![image](https://user-images.githubusercontent.com/122194839/217546475-a080a966-f6a7-48d2-8888-b23b61e45682.png)
you can see the joined picture?

**Reply by tzacha898** [reply]
I have added the rules you asked me for and it still doesn't work

**Reply by mdiego92** [reply]
Hello @tzacha898,

I wanted to check if you were able to add the group and rules just exactly as I sent them. It is important to set up also the overwrite rules to change the eventID they're pointing to.

Thank you!

**Reply by tzacha898** [reply]
Hi @mdiego92 
I've add the rules just exactly as you sent them, also the overwrite rules.
and it still doesn't make any different between disabled user account changed to enable.
I'm not sure you get what I want to do here and I'm not sure if it's possible at all.
I want to get an alert to my pager duty (level 11 and above) just for already disabled user changes to enable.
and I want to separate it from a situation of user has been created,  the problem is when I create a user it is automatically enables him and my wazuh monitors it and cannot separate those situations.
can I overwrite the default rule of this and than change the content to ignore the 4720 624 which means - user has been created?

Thank you so much for your help.

Tzach

**Reply by mdiego92** [reply]
Hello @tzacha898,

Could you please try once again with the following rules? In case this does not work, could you please share with me the content of your `0580-win-security_rules.xml` rule file and your custom rule file to further analyze the issue?

```
<group name="windows,windows_security,">

  <rule id="60109" level="8" overwrite="yes">
    <if_sid>60103</if_sid>
    <if_group>windows</if_group>
    <field name="win.system.eventID">^624$|^4720$</field>
    <options>no_full_log</options>
    <description>User account created.</description>
    <mitre>
      <id>T1098</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>
  
  <rule id="60111" level="8" overwrite="yes">
    <if_sid>60103</if_sid>
    <if_group>windows</if_group>
    <field name="win.system.eventID">^630$|^4726$</field>
    <options>no_full_log</options>
    <description>User account deleted.</description>
    <mitre>
      <id>T1098</id>
      <id>T1531</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>

  <rule id="100111" level="8">
    <if_sid>60103</if_sid>
    <if_group>windows</if_group>
    <field name="win.system.eventID">^626$|^4722$</field>
    <options>no_full_log</options>
    <description>User account enabled.</description>
    <mitre>
      <id>T1098</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>

  <rule id="100112" level="8">
    <if_sid>60103</if_sid>
    <if_group>windows</if_group>
    <field name="win.system.eventID">^629$|^4725$</field>
    <options>no_full_log</options>
    <description>User account disabled.</description>
    <mitre>
      <id>T1098</id>
      <id>T1531</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>
  
</group>
```

Thank you very much.

**Reply by tzacha898** [reply]
Hi @mdiego92 ,
I can't Paste it here as a code because it's huge but i can share my local rules file.
this file contain only this rules we are trying to add.

```

<group name="windows,windows_security,">

  <rule id="60109" level="8" overwrite="yes">
    <if_sid>60103</if_sid>
    <if_group>windows</if_group>
    <field name="win.system.eventID">^624$|^4720$</field>
    <options>no_full_log</options>
    <description>User account created.</description>
    <mitre>
      <id>T1098</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>
  
  <rule id="60111" level="8" overwrite="yes">
    <if_sid>60103</if_sid>
    <if_group>windows</if_group>
    <field name="win.system.eventID">^630$|^4726$</field>
    <options>no_full_log</options>
    <description>User account deleted.</description>
    <mitre>
      <id>T1098</id>
      <id>T1531</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>

  <rule id="100111" level="8">
    <if_sid>60103</if_sid>
    <if_group>windows</if_group>
    <field name="win.system.eventID">^626$|^4722$</field>
    <options>no_full_log</options>
    <description>User account enabled.</description>
    <mitre>
      <id>T1098</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>

  <rule id="100112" level="8">
    <if_sid>60103</if_sid>
    <if_group>windows</if_group>
    <field name="win.system.eventID">^629$|^4725$</field>
    <options>no_full_log</options>
    <description>User account disabled.</description>
    <mitre>
      <id>T1098</id>
      <id>T1531</id>
    </mitre>
    <group>account_changed,adduser,gdpr_IV_32.2,gdpr_IV_35.7.d,gpg13_7.10,hipaa_164.312.a.2.I,hipaa_164.312.a.2.II,hipaa_164.312.b,nist_800_53_AC.2,nist_800_53_AC.7,nist_800_53_AU.14,nist_800_53_IA.4,pci_dss_10.2.5,pci_dss_8.1.2,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
  </rule>
  
</group>
```

**Reply by mdiego92** [reply]
Hello @tzacha898,

It could be possible the alerts are not pointing to the same eventID. Could you please share a screenshot of what alerts are triggered when you enable a deactivated user?

Thank you very much for your patience.

**Reply by tzacha898** [reply]
Hi @mdiego92 
it is the same EVENTID I'm sending you the alerts

![image](https://user-images.githubusercontent.com/122194839/220624612-8c9e4e6b-2223-4591-bf9e-d06ece5f98f8.png)

**Reply by mdiego92** [reply]
Hello @tzacha898,

I'm performing some tests regarding the EventIDs configuration from the ossec.conf file, to see if it's possible to set it up from there after modifying the rules.

I'll update this issue as soon as possible.

Thank you for your patience.

**Reply by mdiego92** [reply]
Hello @tzacha898,

After performing some tests I found out that it is not possible to receive only the User account created. notification, as the AD events are configured by default to create the user and then enable it, and disable it before deleting the account.

As a consequence, Wazuh is receiving the events ID 624, 4720, 626, and 4722 when creating the user, thus enabling it. And the same procedure is reproduced when deleting an account with the following events ID:  630, 4726, 629, and 4725.

It may be possible to change this behavior by changing the auditing settings from the Windows AD configuration, for example, setting a condition to avoid sending events ID 626 and 4722 after creating a user, or something like that. But at the moment, it is not possible to do it from the Wazuh Configuration end, unless you want to block specific events ID.

Please let me know if you have any remaining questions. Sorry for the inconvenience and thank you for your patience.

Regards.
